### PR TITLE
Design website updates

### PR DESCRIPTION
**Pull request by spark3-mdb**
## ✍️ Proposed changes

🎟 _Jira tickets:_ 
[PD-1703](https://jira.mongodb.org/browse/PD-1703) - Badge Guideline Updates
[PD-1789](https://jira.mongodb.org/browse/PD-1789) - Link to LG Github
[PD-1789](https://jira.mongodb.org/browse/PD-1815) - Global direct linking on headings
[PD-1721](https://jira.mongodb.org/browse/PD-1721) - Form guideline link to fixed footer section
[PD-1306](https://jira.mongodb.org/browse/PD-1306) - Allowing Select knob values to be unselected

The following dependencies were also updated:
next: ^10.0.0 -> ^12.1.4
remark: ^12.0.1 -> ^14.0.2
remark-html: ^12.0.0 -> ^15.0.1
remark-parse: 8.0.3 -> ^10.0.1

## 🛠 Types of changes

- [x] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: website/styles/globals.tsx [hunk 1]
```diff
@@ -185,4 +186,28 @@ export const globalStyles = css`
...
+      left: 100%;
+      top: 2px;
+      margin-left: 8px;
+    }
+  }
+
+  li {
+    margin: 12px 0;
+    line-spacing: 1.6;
+  }
```
Comment (spark3-mdb) [inline website/styles/globals.tsx:212]: We REALLY should have a `<List>` component in our library.

File: website/pages/component/confirmation-modal/guidelines.mdx [hunk 2]
```diff
@@ -1,12 +1,10 @@
-import { css } from '@emotion/css';
-import { Body, Link } from '@leafygreen-ui/typography';
+import { css } from "@emotion/css";
```
Comment (shaneeza) [inline website/pages/component/confirmation-modal/guidelines.mdx:1]: We're using single quotes everywhere else, we should probably stick to those. I've noticed there are a few files where they've been switch to double quotes.

File: website/styles/globals.tsx [hunk 3]
```diff
@@ -185,4 +185,28 @@ export const globalStyles = css`
...
+    position: relative;
+
+    &:hover:after {
+      content: url(/connect.svg);
+      position: absolute;
+      left: 100%;
+      top: 2px;
+      margin-left: 8px;
+    }
+  }
```
Comment (TheSonOfThomp) [inline website/styles/globals.tsx:206]: I feel like this icon itself should be clickable

File: website/.babelrc [hunk 4]
```diff
@@ -1,4 +1,4 @@
 {
-  "presets": [["next/babel"], "@emotion/babel-preset-css-prop"],
+  "presets": [["next/babel"], ["@babel/preset-react", { "runtime": "automatic" }]],
```
Comment (TheSonOfThomp) [inline website/.babelrc:2]: Not sure where to add this comment, but it looks like the prop tables disappeared
Screen Shot 2022-04-11 at 2 54 28 PM

**Review by TheSonOfThomp — CHANGES_REQUESTED** [review]
Mostly looks good, though I'm not seeing the prop tables. Is that just an error on my end?

File: website/.babelrc (hunk 4 above)
Comment (spark3-mdb) [reply website/.babelrc:2]: I never noticed that `remark` might have [stopped](https://github.com/remarkjs/remark/discussions/635#discussioncomment-390722) supporting `table` markdown types from <=12 onwards. I've added `remark-gfm` as a plugin to bring the functionality back here.

**Review by shaneeza — APPROVED** [review]
I can see the props table! Looks good. 

Is it possible to get better quality images for the badge design guidelines? It's so pixilated.

**Comment by spark3-mdb** [comment]
> I can see the props table! Looks good.
> 
> Is it possible to get better quality images for the badge design guidelines? It's so pixilated.

Updated PNGs to SVGs!

**Review by TheSonOfThomp — APPROVED** [review]
🍾

**Pull request merged**